submit-api/migrations/versions/97de805275ec_.py

```diff
@@ -0,0 +1,56 @@
+"""empty message
+
+Revision ID: 97de805275ec
+Revises: 6dfc3501adbb
+Create Date: 2024-07-30 10:49:57.735159
+
+"""
+from datetime import datetime
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '97de805275ec'
+down_revision = '6dfc3501adbb'
+branch_labels = None
+depends_on = None
+
+metadata = sa.MetaData()
+roles_table = sa.Table('roles', metadata,
+                       sa.Column('id', sa.Integer, primary_key=True, autoincrement=True),
+                       sa.Column('role_name', sa.String(50), nullable=False),
+                       sa.Column('description', sa.Text, nullable=False),
+                       sa.Column('created_date', sa.DateTime, nullable=False, default=datetime.utcnow),
+                       sa.Column('updated_date', sa.DateTime, nullable=True),
+                       sa.Column('created_by', sa.String(50), nullable=True),
+                       sa.Column('updated_by', sa.String(50), nullable=True)
+                       )
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.bulk_insert(
+        roles_table,
+        [
+            {
+                'role_name': 'ACCOUNT_PRIMARY_ADMIN',
+                'description': 'Administrator role',
+                'created_date': datetime.utcnow(),
+                'created_by': 'system'  # Or replace with the actual creator
```

Comment:
What do you think of adding rest of the roles if you already know them like PROJECT_ADMIN

Reply:
let's agree on the list and I can follow that up in another PR